integrationTests/integrations.yml

```diff
@@ -17,7 +17,7 @@ invalidVote:
   voteCreditBalance: 1
 constants:
   poll:
-    duration: 120
+    duration: 250
```

Comment:
Why? Was this timing out?

Reply:
Yes seems like sometimes the poll duration was not enough, hence why it was increased.

Reply:
@ctrlc03 Could you please clarify where these timeout issues were occurring? Was it in a local environment or during the CI process? If it's the latter, could you provide a link to the relevant CI build or log?